app/views/exam_templates/_form.html.erb

```diff
@@ -14,6 +14,20 @@
       <p><%= raw t('assignment.required_fields') %> <span class='required_field'>*</span></p>
       <%= f.label t('exam_templates.update.instruction') %>
       <%= f.file_field :new_template %>
-      <p><%= submit_tag t(:submit)%></p>
     </fieldset>
+
+    <fieldset>
+      <legend><%= ExamTemplate.human_attribute_name('template_divisions') %></legend>
+      <p><strong><%= t('exam_templates.update.template_divisions') %></strong></p>
+      <p><%= raw t('assignment.required_fields') %> <span class='required_field'>*</span></p>
+      <% @select = Array.new(exam_template.num_pages) { |index| index + 1 }%>
+      <%= f.fields_for :template_divisions do |form| %>
+        <p>
+          <%= form.text_field :label, required: true %>
+          <%= form.number_field(:start) %>
```

Comment:
Good - please pass in the necessary options to ensure that these inputs are >= 1.